src/timesync/timesyncd.c

```diff
@@ -132,6 +132,12 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 log_warning_errno(r, "Failed to parse configuration file: %m");
 
+        r = manager_parse_fallback_string(m, NTP_SERVERS);
+        if (r < 0) {
+                log_error_errno(r, "Failed to parse fallback server configuration: %m");
+                goto finish;
+        }
```

Comment:
I'd just do an `assert_se(m_p_f_s() >= ))`, and also add a unit test that this works. This will be better, because in case the distro screws up and NTP_SERVERS is wrong, it'll be detected at compilation time.